skia/RNSVGSvgViewModule.cpp

```diff
@@ -19,6 +19,7 @@ std::string RNSVGSvgViewModule::getName() {
 }
 
 auto RNSVGSvgViewModule::getConstants() -> std::map<std::string, folly::dynamic> {
+  RNS_LOG_NOT_IMPL;
```

Comment:
I dont think there is any constants required here right ? So it should just return empty object. No need of not implemented warning.